### PR TITLE
Use message bar; no dialog for user to close

### PR DESCRIPTION
**Pull request by AlisterH**
Dialogs are old school, so they say.

### CODE REVIEW

**Comment by raymondnijssen** [comment]
LOL, I just created the exact same PR. I think we had the same annoyance...

File: QCopycanvas.py [hunk 1]
```diff
@@ -185,25 +186,11 @@ def unload(self):
...
     def run(self):
         """Run method that performs all the real work"""
 
         # Create the dialog with elements (after translation) and keep reference
         # Only create GUI ONCE in callback, so that it will only load when the plugin is started
         QApplication.clipboard().setImage(QImage(QWidget.grab(self.iface.mapCanvas())))
-        self.showMessage(title='QCopycanvas', msg='Copied map canvas successfully.', button='OK')
\ No newline at end of file
+        #self.showMessage(title='QCopycanvas', msg='Copied map canvas successfully.', button='OK')
+        self.iface.messageBar().pushMessage('QCopycanvas','Copied map canvas to clipboard', level=Qgis.Success,)
```
Comment (raymondnijssen) [inline QCopycanvas.py:196]: What is that last comma in the pushMessage() function doing? Should it have a duration too?

**Comment by AlisterH** [comment]
I guess I left the comma by mistake.  I figured there is no reason not to let it use the default duration.

**Comment by Mariosmsk** [comment]
8f4fb6de5a87e1513b6873dd6cef2c32d25a381e

**Pull request closed without merging**